app/javascript/dashboard/store/modules/contactLabels.js

```diff
@@ -0,0 +1,88 @@
+import Vue from 'vue';
+import * as types from '../mutation-types';
```

Comment:
Please use `import types from '../mutation-types` and `types.SET_CONTACT_LABELS_UI_FLAG` instead of `types.default.SET_CONTACT_LABELS_UI_FLAG`